tests/integration/tier1/kernel-boot-files/test_handle_corrupted_files.py

```diff
@@ -0,0 +1,72 @@
+import os
+import shlex
+import subprocess
+
+from multiprocessing import Pool
+
+import pytest
+
+from envparse import env
+
```

Comment:
```suggestion
from conftest import SYSTEM_RELEASE_ENV
```
Let's keep it consistent with #653 please.